elasticsearch/src/test/java/com/tcdi/zombodb/query_parser/TestQueryRewriter.java

```diff
@@ -148,6 +148,17 @@ public Map answer(InvocationOnMock invocationOnMock) throws Throwable {
                             fieldProps.put("analyzer", "fulltext");
 
                             fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+
+                            fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+                            final Map fieldsMap = new HashMap();
+                            fieldsMap.put("date", "something about date");
+                            fieldProps.put("fields", fieldsMap);
```

Comment:
Yeah, all of the tally() tests have been from the SQL tests.  This is good tho.

I actually hate this .java file.  I clearly spent zero effort making it maintainable.  And at the time I didn't know how to bootstrap an ES node through Java, but I do now, so I might (eventually) get around to reworking this stuff.